{"name": "test_create_account", "status": "failed", "statusDetails": {"message": "AssertionError: Expected status code 201 but got 200", "trace": "api_client = <api.api_client.APIClient object at 0x7274077b0380>\n\n    @allure.epic(\"User Management\")\n    @allure.feature(\"Create Account\")\n    @allure.story(\"Creating a new user account\")\n    @allure.severity(allure.severity_level.CRITICAL)\n    def test_create_account(api_client):\n        with allure.step(\"Создание данных для запроса\"):\n            data = CreateAccountRequest.get_default().model_dump()\n    \n        with allure.step(\"Отправка запроса на создание аккаунта\"):\n            response = api_client.create_account(data)\n    \n        with allure.step(\"Проверка кода статуса ответа\"):\n>           assert_status_code(response, 201)\n\ntests/test_api/test_createaccount.py:18: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresponse = <Response [200]>, expected_status_code = 201\n\n    def assert_status_code(response, expected_status_code):\n>       assert response.status_code == expected_status_code, \\\n            f\"Expected status code {expected_status_code} but got {response.status_code}\"\nE       AssertionError: Expected status code 201 but got 200\n\napi/assertions.py:2: AssertionError"}, "steps": [{"name": "Создание данных для запроса", "status": "passed", "start": 1731168321782, "stop": 1731168321783}, {"name": "Отправка запроса на создание аккаунта", "status": "passed", "start": 1731168321783, "stop": 1731168322698}, {"name": "Проверка кода статуса ответа", "status": "failed", "statusDetails": {"message": "AssertionError: Expected status code 201 but got 200\n", "trace": "  File \"/home/dadyc1own/PycharmProjects/Site_Automationexercise/tests/test_api/test_createaccount.py\", line 18, in test_create_account\n    assert_status_code(response, 201)\n  File \"/home/dadyc1own/PycharmProjects/Site_Automationexercise/api/assertions.py\", line 2, in assert_status_code\n    assert response.status_code == expected_status_code, \\\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n"}, "start": 1731168322698, "stop": 1731168322698}], "start": 1731168321782, "stop": 1731168322705, "uuid": "50b059ee-0597-4ec5-8bc5-108015cd04c5", "historyId": "948afa17ad818c22abb96e47d377eed9", "testCaseId": "948afa17ad818c22abb96e47d377eed9", "fullName": "tests.test_api.test_createaccount#test_create_account", "labels": [{"name": "epic", "value": "User Management"}, {"name": "story", "value": "Creating a new user account"}, {"name": "severity", "value": "critical"}, {"name": "feature", "value": "Create Account"}, {"name": "parentSuite", "value": "tests.test_api"}, {"name": "suite", "value": "test_createaccount"}, {"name": "host", "value": "dadyc1own-laptop"}, {"name": "thread", "value": "119194-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_api.test_createaccount"}]}